M2/Macaulay2/packages/SimpleDoc/helpers-doc.txt

```diff
@@ -0,0 +1,81 @@
+-- -*- mode: M2; coding: utf-8 -*-
+Node
+  Key
+     wikipedia
+    (wikipedia, String)
+    (wikipedia, String, String)
+  Headline
```

Comment:
How would this be used in a doc string?

Reply:
I'll mention an example in the documentation for `doc(String)`.

Reply:
Why is "wikipedia" in the SimpleDoc package now?  It has nothing to do with it, so why move it there from Core?  Similarly for the other ones.

Reply:
In this package people are a lot more likely to find the documentation than in Macaulay2Doc. I considered doing the same thing as the Text package, which only contains documentation and no definitions, but `wikipedia` is a single line and never used in Core, so there's no point in it being there either.